ci-operator/config/openshift/kuryr-kubernetes/openshift-kuryr-kubernetes-master.yaml

```diff
@@ -61,6 +61,14 @@ tests:
   commands: HOME=`dirname .` tox -e pep8
   container:
     from: kuryr-tester
+- as: e2e-openstack-kuryr
+  steps:
+    cluster_profile: openstack-kuryr
+    env:
+      NETWORK_TYPE: Kuryr
+      OPENSTACK_COMPUTE_FLAVOR: m1.xlarge
+      OPENSTACK_EXTERNAL_NETWORK: external
```

Comment:
Flavor and ext-network for `openstack-kuryr` are already set [here](https://github.com/openshift/release/blob/d5bd10db63fc35b952bed6f47f62ef78121402e2/ci-operator/step-registry/openstack/conf/resourcenames/openstack-conf-resourcenames-commands.sh#L6-L13) with the same values: there should be no need to override.